packages/nextjs/components/scaffold-eth/Contract/utilsContract.tsx

```diff
@@ -115,30 +119,32 @@ const getContractReadOnlyMethodsWithParams = (
  * which has corresponding input field for param type, txnValue input if required and button to send transaction
  */
 const getContractWriteMethods = (
-  contract: Contract,
+  contract: Contract | null,
   contractMethodsAndVariables: FunctionFragment[],
   setRefreshDisplayVariables: Dispatch<SetStateAction<boolean>>,
 ): { loaded: boolean; methods: (JSX.Element | null)[] } => {
-  return {
-    loaded: true,
-    methods: contractMethodsAndVariables
-      .map((fn, index) => {
-        const isWriteableFunction = fn.stateMutability !== "view" && fn.stateMutability !== "pure";
-        if (isWriteableFunction) {
-          // FFW -> FunctionFormWrite
-          return (
-            <WriteOnlyFunctionForm
-              key={`FFW_${fn.name}_${index}`}
-              functionFragment={fn}
-              contractAddress={contract.address}
-              setRefreshDisplayVariables={setRefreshDisplayVariables}
-            />
-          );
-        }
-        return null;
-      })
-      .filter(n => n),
-  };
+  return contract
+    ? {
+        loaded: true,
+        methods: contractMethodsAndVariables
+          .map((fn, index) => {
+            const isWriteableFunction = fn.stateMutability !== "view" && fn.stateMutability !== "pure";
+            if (isWriteableFunction) {
+              // FFW -> FunctionFormWrite
+              return (
+                <WriteOnlyFunctionForm
+                  key={`FFW_${fn.name}_${index}`}
```

Comment:
and here

Reply:
check https://github.com/scaffold-eth/se-2/pull/96#discussion_r1080773581